app/profile/EditProfile.tsx

```diff
@@ -1,23 +1,43 @@
 import React from "react";
-import { Image, ScrollView, StyleSheet, TouchableWithoutFeedback, View } from "react-native";
+import { Image, Keyboard, ScrollView, StyleSheet, TouchableWithoutFeedback, View } from "react-native";
 import { router } from "expo-router";
 import { Feather } from "@expo/vector-icons";
-import { wp } from "@/utils";
+import { EditProfileI } from "@/interfaces";
 import { useAuthStore } from "@/store";
+import { useFormikHook } from "@/hooks";
+import { editProfileValidationSchema, wp } from "@/utils";
 import { colorPalette, LayoutStyles, Spacing } from "@/styles";
 import { AppButton, AppHeader, GradientWrapper, TextInput } from "@/components";
 import profilePicture from "../../assets/images/Profile10.png";
```

Comment:
Replace this with Absolute path in all screens, components. This shouldn't repeat again in anywhere in code!!!